viewer/src/public/migration/Cognite3DViewer.ts

```diff
@@ -56,6 +56,7 @@ import { CadIntersection, IntersectionFromPixelOptions, PointCloudIntersection }
  * @module @cognite/reveal
  */
 export class Cognite3DViewer {
+  renderTarget: THREE.WebGLRenderTarget | undefined;
```

Comment:
```suggestion
```
Unused?

Reply:
Odd that the linter didn't complain here. When I don't format with correct whitespacing I get error, but unused vars is whatever apparently...